src/index.ts

```diff
@@ -49,23 +54,41 @@ export type PermitProps = {
   userAttributes?: Record<string, any>;
   backendUrl: string;
   defaultAnswerIfNotExist?: boolean;
+  customRequestHeaders?: AxiosRequestHeaders;
 };
 
-const getBulkPermissionFromBE = async (url: string, user: string, actionsResourcesList: ActionResourceSchema[]): Promise<boolean[]> => {
+const getBulkPermissionFromBE = async (
+  url: string,
+  user: string,
+  actionsResourcesList: ActionResourceSchema[],
+  headers?: AxiosRequestHeaders,
+): Promise<boolean[]> => {
+  console.log("making request")
+
   const payload = actionsResourcesList.map(({ action, resource, userAttributes = {}, resourceAttributes = {} }) => ({
     action,
     resource: typeof resource === 'string' ? resource : `${resource.type}:${resource.key}`,
     userAttributes,
     resourceAttributes,
   }));
+  const config = headers ? { headers } : {};
 
-  const response = await axios.post(`${url}?user=${user}`, { resourcesAndActions: payload });
+  const response = await axios.post(`${url}?user=${user}`, { resourcesAndActions: payload }, config);
   return response.data.permittedList;
 };
 
-const getPermissionFromBE = async (url: string, user: string, action: string, resource: string, defaultPermission: boolean): Promise<boolean> => {
+const getPermissionFromBE = async (
+  url: string,
+  user: string,
+  action: string,
+  resource: string,
+  defaultPermission: boolean,
+  headers?: AxiosRequestHeaders,
+): Promise<boolean> => {
+  console.log("making request")
```

Comment:
you can remove it

Reply:
Ooops. Good ole reliable console.log 😅
Thanks for catching it